src/Symfony/Component/Messenger/Stamp/RoutingKeyStamp.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Messenger\Stamp;
```

Comment:
A "routing key" is very Amqp specific, I think the stamp should go to the `AmqpExt` namespace.